README.md

```diff
@@ -148,6 +136,66 @@ var nuxeo = new Nuxeo({
 });
 ```
 
+### Testing the Connection and Retrieve the Current User
+
+After creating a Client, you can test the connection by using the `login` method which resolve
+with the current user.
+
+```javascript
+var nuxeo = new Nuxeo({ ... });
+nuxeo.login()
+  .then(function(user){
+    // user is a full User object
+    // nuxeo.user === user
+    console.log(user);
+  })
+  .catch(function(error) {
+    // wrong credentials / auth method / ...
+    throw error;
+  });
+```
+
+Note that `login` also fill the `user` property of the client.
+
+### Checking Nuxeo Server version
+
+Since the 3.0 version, the JS Client may expose the Nuxeo Server version, so that users of the client
+can make choices depending on the Nuxeo version.
+
+This Nuxeo Server version is retrieved when using the `login` method on the Client which fills
+the `nuxeoVersion` property of the client.
```

Comment:
Since version 3.0, the JS Client gives access to the Nuxeo server version: it is retrieved in the `nuxeoVersion` property after call of the `login()` method.